### DISCUSSION TITLE
¿Is possible to set a max Cluster size?

### DISCUSSION THREAD

**Discussion by Manuframil** [moj-analytical-services/splink#2254] [Q&A]
Im merging users from 2 data sources, and users can appear in one of them or both, but not twice in the same data source, so I want to be able to force a max cluster size of 2, so Splink doesnt unify users that shouldnt be unified (because they already have a better match). 

For example, this 4 users should be clustered in two seperated sets, so the higher match should be between Manuel Rodriguez (unique_id=149) and Manuel Rodríguez (unique_id=150), which is almost an exact match. Then, the other two M. and Miguel, should be in another cluster.
![Screenshot_20240715_190813](https://github.com/user-attachments/assets/cd4d9650-2bfb-4329-a41f-69c1ecdcfffc)

¿is this possible with Splink? ¿Am I poiting in the right direction, or should I look for a different approach?

Thanks in advance!

**Comment by RobinL** [comment]
You can't set a max cluster size.

This post may be of interest/use:
https://github.com/moj-analytical-services/splink/issues/251
